packages/test/test-end-to-end-tests/src/test/sharedStringLoading.spec.ts

```diff
@@ -132,7 +132,13 @@ describeNoCompat("SharedString", (getTestObjectProvider) => {
 					},
 				};
 				const codeDetails = { package: "no-dynamic-pkg" };
-				const codeLoader = new LocalCodeLoader([[codeDetails, fluidExport]]);
+				const codeLoader = new LocalCodeLoader([[codeDetails, fluidExport]], {
+					summaryOptions: {
+						summaryConfigOverrides: {
+							state: "disabled",
```

Comment:
```suggestion
							// disable the summarizer to prevent the above fault injection from happening
							// happening in the summarizer client
							state: "disabled",
```